tools/ci.psm1

```diff
@@ -593,3 +584,176 @@ function Invoke-AppveyorFinish
         throw $_
     }
 }
+
+function Invoke-LinuxTests
+{
+    $createPackages = $isFullBuild
```

Comment:
you need to move the below where `$isFullBuild` is defined.

Reply:
Okay